test/viewmodels/helpers/CheckAnswersConsigneeHelperSpec.scala

```diff
@@ -167,13 +176,24 @@ class CheckAnswersConsigneeHelperSpec extends SpecBase {
 
           val expectedSummaryListRows: Seq[SummaryListRow] = Seq(
             ConsigneeBusinessNameSummary.row(true)(fakeDataRequest, msgs),
-            ConsigneeExciseSummary.row(true)(fakeDataRequest, msgs),
+            consigneeExciseSummary.row(true)(fakeDataRequest, msgs),
             ConsigneeAddressSummary.row(true)(fakeDataRequest, msgs)
           ).flatten
 
           checkAnswersConsigneeHelper.summaryList() mustBe SummaryList(rows = expectedSummaryListRows)
         }
 
+      "the Consignee ERN needs to be updated" in new Setup(testGbWarehouseErn, ernNeedsUpdatingUserAnswers) {
+
+        val expectedSummaryListRows: Seq[SummaryListRow] = Seq(
+          ConsigneeBusinessNameSummary.row(true)(fakeDataRequest, msgs),
+          consigneeExciseSummary.row(true)(fakeDataRequest, msgs),
+          ConsigneeAddressSummary.row(true)(fakeDataRequest, msgs)
+        ).flatten
+
+        checkAnswersConsigneeHelper.summaryList() mustBe SummaryList(rows = expectedSummaryListRows)
+      }
```

Comment:
I don't think this is providing much value, is there a way we can check that consigneeExciseSummary has an update needed tag?